interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.ts

```diff
@@ -35,16 +36,32 @@ export class DisasterTypeComponent implements OnInit, OnDestroy {
     this.countrySubscription.unsubscribe();
   }
 
+  private onGetDisasterTypeActiveTrigger = (country) => () => {
+    if (this.disasterTypesCounter >= country.disasterTypes.length - 1) {
+      const activeDisasterType = country.disasterTypes.find(
+        ({ activeTrigger }) => activeTrigger,
+      );
+      if (activeDisasterType) {
+        this.selectedDisasterType =
+          DisasterTypeKey[activeDisasterType.disasterType];
+        this.disasterTypeService.setDisasterType(activeDisasterType);
+      } else {
+        this.selectedDisasterType = this.disasterTypes[0].disasterType;
+      }
+    }
+    this.disasterTypesCounter++;
+  };
+
   private onCountryChange = (country: Country) => {
     if (country) {
       this.disasterTypes = country.disasterTypes;
       this.disasterTypes.forEach((disasterType) => {
```

Comment:
@SwatiTh Maybe you should before this line order the disasterTypes (alphabetically) to make sure they're always in the same order.